_content/tour/por/slices.article

```diff
@@ -1,29 +1,31 @@
 Slices
-Slices are an incredibly important data structure in Go. They form the basis for how we manage and manipulate data in a flexible, performant, and dynamic way. It's very beneficial for all Go programmers to learn how slices work, and how to use them.
+`Slices` são uma estrutura de dados incrivelmente importante em Go. Elas formam a base de como gerenciamos e
+manipulamos dados de forma flexível, eficiente e dinâmica. É muito benéfico para todos os programadores
+em Go aprenderem como `Slices` funcionam e como usá-los.
```

Comment:
The section description needs to be in one single line or the program returns a parse error.

```suggestion
`Slices` são uma estrutura de dados incrivelmente importante em Go. Elas formam a base de como gerenciamos e manipulamos dados de forma flexível, eficiente e dinâmica. É muito benéfico para todos os programadores em Go aprenderem como `Slices` funcionam e como usá-los.
```